charts/akeyless-api-gateway/templates/cache.yaml

```diff
@@ -0,0 +1,101 @@
+{{- if eq (include "akeyless-api-gw.adminAccessUidExist" .) "true" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: {{ include "akeyless-api-gw.fullname" . }}-cache
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+{{- include "akeyless-api-gw.labels" . | nindent 4 }}
+    name: {{ template "akeyless-api-gw.fullname" . }}-cache
+    component: cache
+  name: {{ template "akeyless-api-gw.fullname" . }}-cache
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: {{ template "akeyless-api-gw.fullname" . }}-cache
+  template:
+    metadata:
+      labels:
+        name: {{ template "akeyless-api-gw.fullname" . }}-cache
+        component: cache
+    spec:
+      containers:
+      - name: {{ template "akeyless-api-gw.fullname" . }}-cache
+        image: docker.io/bitnami/redis:6.2 
+        imagePullPolicy: Always
+        #args: ["--requirepass", "$(REDIS_PASSWORD)"]
+        ports:
+        - containerPort: 6379
+          name: cache
+        livenessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - |            
+                #!/usr/bin/env bash -e
```

Comment:
from where did you took this base script? can you add a link reference

Reply:
https://stackoverflow.com/questions/68304279/incorrect-liveness-probe-for-redis-not-failing